tutorial/.gitattributes

```diff
@@ -0,0 +1,2 @@
+*   text=auto
+*   eol=lf
```

Comment:
We're developing in Linux so I prefer to set all my line endings to LF. Optional.

Reply:
This is based on the assumption that VSCode is going to be used by everyone following this tutorial. If we don't use it, then it becomes quite an issue.